scripts/download-integration-test-binaries.sh

```diff
@@ -38,31 +45,113 @@ function download() {
 	fi
 	echo ">>>"
 	echo "download ${file_name} from ${url}"
-	wget --no-verbose --retry-connrefused --waitretry=1 -t 3 -O "${file_path}" "${url}"
+	curl --retry 3 --retry-delay 1 -o "${file_path}" "${url}"
 }
 
-# Specify the download branch.
-branch=$1
+# download_community_version will try to download required binaries from the
+# public accessible community version
+function download_community_binaries() {
+	set +u
+	local ver=${1:-v6.5.2}
+	local os=${2:-linux}
+	local arch=${3:-amd64}
+	set -u
+	local dist="${ver}-${os}-${arch}"
+	local tidb_file_name="tidb-community-server-$dist"
+	local tidb_tar_name="${tidb_file_name}.tar.gz"
+	local tidb_url="https://download.pingcap.org/$tidb_tar_name"
+	local toolkit_file_name="tidb-community-toolkit-$dist"
+	local toolkit_tar_name="${toolkit_file_name}.tar.gz"
+	local toolkit_url="https://download.pingcap.org/$toolkit_tar_name"
+
+	color-green "Download community binaries..."
+	download "$tidb_url" "$tidb_tar_name" "tmp/$tidb_tar_name"
```

Comment:
Which version it will download?

Reply:
for now, 6.5.2 will be downloaded.

Reply:
The latest update includes a new commit that enables users to specify the version (default: 6.5.2), operating system (default: Linux), and architecture (default: amd64).

Reply:
Does the `version` conflict with the original `branch` parameter?